### DISCUSSION TITLE
opern5gs rename to XXX

### DISCUSSION THREAD

**Discussion by Sivanesh1992** [herlesupreeth/docker_open5gs#203] [General]
hi @herlesupreeth  i am trying to rename the docker_open5gs to docker_open5gs_master in  core network

i am getting below error .

docker_open5gs_master$ docker-compose up -d
Creating network "docker_open5gs_master_default" with the default driver
ERROR: Pool overlaps with other one on this address space

is possible to rename the folder and up the containers?

**Comment by herlesupreeth** [comment]
yes, it should be possible

> docker_open5gs_master$ docker-compose up -d
> Creating network "docker_open5gs_master_default" with the default driver
> ERROR: Pool overlaps with other one on this address space

You need to change the subnet of the docker_open5gs network. There is already a network created with subnet you are trying to use.
